events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -115,10 +117,15 @@
         fillRandomSoundsArray();
     }
 
-    /* FIND OR CREATE BINGO CARD:  Search the google sheet for the user's name to see if they have already been assigned 
-    numbers for this round. If they have numbers already, retrieve them. If they do not have numbers, get new ones
-    and add one to the player counter text. */
-    var SPREADSHEET_URL = Script.require(Script.resolvePath('../secrets/bingoSheetURL.json?0')).sheetURL;
+    // Two main cases here:
+    // 1. The user has previously filled their `userCardNumbers` array with a result from the backend.
+    //     Send a message to the UI with those numbers, the selectedNumberIDs, and the cardColor
+    // 2. The user doesn't have a cached `userCardNumbers` array. In that case:
+    //     i. Request card numbers and card color from the server
+    //     ii. If the response indicates success, send that data to the UI
+    //         a. If the server indicates that the user is new, tell the player counter text to
+    //             add one to its count.
+    var REQUEST_URL = Script.require(Script.resolvePath('../config/config.json?' + Date.now())).requestURL;
```

Comment:
Similar question with Date.now()

Reply:
Really got to figure out what the function is to clear the module cache.  can't remember if I ever got this working: https://docs.highfidelity.com/api-reference/namespaces/script#.resetModuleCache

Reply:
which question do you have here?

Reply:
Just about using Date.now() for caching modules and should that be in production code.

Reply:
ah yes! that's okay to do.